cli/fathom_web/extract.py

```diff
@@ -83,7 +83,7 @@ def extract_base64_data_from_html_page(file: pathlib.Path):
         html = fp.read()
 
     # Make the subresources directory
-    subresources_directory = file.parent / 'resources' / f'{file.stem}_resources'
+    subresources_directory = file.parent / f'{file.stem}_resources'
```

Comment:
Oh, I think we had a miscommunication. I thought the outcome of yesterday's conversation was that we'd keep the `samples/negatives/resources/45/2.png`-like layout and just remove the "_resources" suffix.

Reply:
Indeed we did. I think this stems out of how we think `fathom-pick` will work now. Will `fathom-pick` create a `resources` directory in your destination directory and move the page-specific resource directories for the randomly picked items into that new `resources` directory?

Reply:
Yep. And if the dir is already there, it'll just add stuff to it. But if the sample-specific subdir is already there, it should error.

Reply:
Gotcha. I'm on it!